src/util/Options.js

```diff
@@ -58,6 +58,11 @@ class Options {
                     },
                 },
                 useragent: null
+            },
+            cache: {
+                enabled: true,
+                ttl: 10,
+                leaderboardttl: 1
```

Comment:
As I said in Discord, we could make this a Map or Object to map TTL to managers. Not the most important thing right now though.

Reply:
I think it's better to put these configurations like this because in any case we'll be able to modify it when we instantiate the Client class. I'll see later if we need to change it

Reply:
Sure, I'm cool with that 👍